### PR TITLE
Tax Counseling for the Elderly eligibility

### LINKED ISSUE #4751: Tax Counseling for the Elderly eligibility
[Reference](https://www.irs.gov/individuals/tax-counseling-for-the-elderly) - we can model payout separately

### PR DESCRIPTION
**Pull request by vrathi101**
Fixes #4751

### CODE REVIEW

File: policyengine_us/variables/gov/irs/tce/is_tce_eligible.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/irs/tce/is_tce_eligible.py:1]: would we want to incorporate this into the head start PR?

Comment (vrathi101) [reply policyengine_us/variables/gov/irs/tce/is_tce_eligible.py:1]: > would we want to incorporate this into the head start PR?

i don't think so(?), not exactly related

**Comment by PavelMakarchuk** [comment]
@MaxGhenis how would we want to incorporate this into the net income tree?

**Comment by MaxGhenis** [comment]
Not until we attach values

**Comment by vrathi101** [comment]
@PavelMakarchuk should we leave it as this then?

**Review by PavelMakarchuk — APPROVED** [review]

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]
Could you take a look through [this](https://www.irs.gov/pub/irs-pdf/p1101.pdf) manual, there might be some income limits

**Comment by vrathi101** [comment]
> Could you take a look through [this](https://www.irs.gov/pub/irs-pdf/p1101.pdf) manual, there might be some income limits

none mentioned, most of the manual talks about how to operate the program (similar to head start)

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**